resources/web/wwi/Animation.js

```diff
@@ -458,7 +458,7 @@ export default class Animation {
     document.getElementById('left-pane').appendChild(this._currentTime);
 
     const timeDivider = document.createElement('span');
-    timeDivider.innerHTML = '\\';
+    timeDivider.innerHTML = '\/';
```

Comment:
Isn't it working simply like that?
```suggestion
    timeDivider.innerHTML = '/';
```

Reply:
yes